lib/matplotlib/tests/test_widgets.py

```diff
@@ -118,6 +131,260 @@ def onselect(epress, erelease):
         assert artist.get_alpha() == 0.3
 
 
+def test_rectangle_resize():
+    ax = get_ax()
+
+    def onselect(epress, erelease):
+        pass
+
+    tool = widgets.RectangleSelector(ax, onselect, interactive=True)
+    # Create rectangle
+    _resize_rectangle(tool, 0, 10, 100, 120)
+    assert tool.extents == (0.0, 100.0, 10.0, 120.0)
```

Comment:
It's a bit confusing that the input to `_resize_rectangle` is a different order to that returned by `extents` - could you change the input order of `resize_rectangle` to match?